labshare-client/src/backend/database/schemas/IUserAdmin.ts

```diff
@@ -0,0 +1,31 @@
+import { Schema, Document } from "mongoose";
+import { ITimestamp } from './ITimestamps';
+export interface IUserAdmin extends Document, ITimestamp {
+    password: string,
+    disabled: boolean,
+    contact: {
+        email: string,
+        firstname: string,
+        lastname: string
+    }
+    role: string
```

Comment:
Made a second `UserAdminRole` enum, want to keep them as much as seperated as possible, so that they don't infere with the standard user roles, which might cause some security issues...